xpano/gui/panels/about.cc

```diff
@@ -74,18 +82,26 @@ void AboutPane::Draw() {
   ImGui::Begin("About", &show_, window_flags);
 
   ImGui::SetNextItemWidth(ImGui::GetContentRegionAvail().x);
+
+  if(license_text[1] == DefaultNotice() && IsReady(licenses_)) {
```

Comment:
There is no need to check the equality with default notice, maybe there was some confusion, you can use:
```
if(licenses.size() == 1 && IsReady(licenses_future_)
```